### PR TITLE
Feature: Fibonacci Series

### PR DESCRIPTION
**Pull request by ksrscharan**
## Description
Implementing Fibonacci Series. 
### Changes
- Used Prettier to format code.
- Used a function for fibonacci sequence.
- Updated PR description.

## Database schema changes
NA

### Automated test cases added
NA

#### Test Case 1:
- _input:_ ```0```
- _expected output:_ ```[]```
- _output:_ ```[]```
#### Test Case 2:
- _input:_ ```1```
- _expected output:_ ```[0]```
- _output:_ ```[0]```
#### Test Case 3:
- _input:_ ```6```
- _expected output:_ ```[0, 1, 1, 2, 3, 5]```
- _output:_ ```[0, 1, 1, 2, 3, 5]```

### CODE REVIEW

**Review by sourabh-4097 — CHANGES_REQUESTED** [review]
1. Remove console.log statements
2. Look into prettier extension in VS-Code for code-formatting and format files using the same
3. Have proper function for fibonacci sequesnce and you should be able to pass just number to function which returns fibonacci series
4. Update PR-description which defines what you are exactly doing in the PR
(eg. - Implemented fibonacci series function )
5. Assign PR to yourself and add proper label ( Enhancement in our case )

File: index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    arr.push(n1)
+    nextTerm = n1 + n2;
+    n1 = n2;
+    n2 = nextTerm;
+  }
+  return arr
+}
+const number = parseInt(prompt("enter number"));
+
+fibonacci(number)
```
Comment (sourabh-4097) [inline index.js]: Add new line

File: index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+function fibonacci(number) {
+  let arr = []
+  let n1 = 0,
+    n2 = 1,
```
Comment (sourabh-4097) [inline index.js:4]: let n2

File: index.js (hunk 1 above)
Comment (sourabh-4097) [reply index.js]: Ensure that you are formatting code using prettier, in that case, it adds new line itself

File: index.js (hunk 2 above)
Comment (ksrscharan) [reply index.js:4]: Upon adding comma, let doesn’t have to be present. Prettier is making indentation weird.

**Comment by sourabh-4097** [comment]
Please don't change the PR-description template. Keep it default and place N/A where you think you haven't changed anything

File: index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        arr.push(n1);
+        nextTerm = n1 + n2;
+        n1 = n2;
+        n2 = nextTerm;
+    }
+    
+    return arr;
+}
+
+const number = parseInt(prompt("enter number of terms"));
```
Comment (sourabh-4097) [inline index.js]: you can have message such as ` Please enter number upto which you want fibonnaci series`

File: index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+fibonacci(number);
```
Comment (sourabh-4097) [inline index.js]: Add new-line here

File: index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+function fibonacci(number) {
```
Comment (sourabh-4097) [inline index.js]: you can also have function name as `printFibonacciNumbers`  which sounds more better

**Review by sourabh-4097 — APPROVED** [review]

**Pull request merged**